ds/data/processed/travel/travel_to_airport_time.lep.yaml

```diff
@@ -1,6 +1,7 @@
 api_type: FETCH
 title: Travel time to airport
 subtitle: Minimum travel time to nearest airport from a LEP region for England (2011 & 2013) and Wales (2011 only)
+description: Minimum travel time to nearest airport from a LEP region for England (2011 & 2013) and Wales (2011 only). Note: Overlapping LEP regions are used therefore some calculations may be under-estimated or over-estimated.

```

Comment:
(1) We need to have the `description` value within single apices as the presence of `:` causes an error when running the script updating the data for the app